### PR TITLE
[FIX] Implement Rush Hour after Crafting Food

### PR DESCRIPTION
**Pull request by j923sneaks**
# Description

This PR fixes the issue where crafting the food with Rush Hour doesn't start the countdown at reduced start point. This is because the skill check is only on cook.ts and not in craftingMachine's `assignCraftingDetails` action.

Fixes #issue

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manual test - local

https://user-images.githubusercontent.com/89294757/202638353-4967972a-7f19-4c13-9166-8f4aac9e86bb.mp4

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by j923sneaks** [comment]
lightning icon currently in discussion #1693

**Review by Rofrtd — CHANGES_REQUESTED** [review]
Looking good, just left a comment for the test you've updated!

File: src/features/game/events/landExpansion/cook.test.ts [hunk 1]
```diff
@@ -257,10 +257,15 @@ describe("cook", () => {
 describe("getReadyAt", () => {
   it("applies 20% speed boost with Rush Hour skill", () => {
     const now = Date.now();
+    const bumpkin = GAME_STATE.bumpkin;
```
[the remark this answers is not in the document]
Comment (j923sneaks) [reply src/features/game/events/landExpansion/cook.test.ts]: applied. thanks

**Review by Rofrtd — APPROVED** [review]
Great job mate!

LGTM.

**Pull request merged**